### ISSUE TITLE
Highlighting of strings ("...") not working correctly

### ISSUE THREAD

**Issue by Dev-iL** [Gimly/vscode-matlab#53]
The MATLAB editor colors text delimited by single quotes (`'`) the same way it does with double quotes (`"`). This is not the case when using the plugin:

![image](https://user-images.githubusercontent.com/6509619/40362508-61035ee8-5dd5-11e8-822d-9e4a05d8ab7f.png)

This is especially confusing since something red usually indicates invalid syntax, which is clearly not the case here.

**Comment by sco1** [comment]
For reference, MATLAB's highlighting of:

```
A = 'sometext';
B = "sometext";
! This should not look the same as a text or a comment
```

<img width="507" alt="matlab" src="https://user-images.githubusercontent.com/5323929/40682344-7728f26c-6359-11e8-84e2-d01a1ee0adfd.png">

MATLAB's IDE has a discrete color for system commands (lines that begin with `!`):

<img width="535" alt="colors" src="https://user-images.githubusercontent.com/5323929/40682487-e3ac481c-6359-11e8-9b1a-7bb0a8f636c8.png">

**Comment by psychelzh** [comment]
Is there any news about this? I do think this is a very significant bug to fix.

**Comment by Gimly** [comment]
@psychelzh This issue has been fixed by version 1.1.0. If you feel this hasn't been fully corrected or there is another issue, feel free to reopen.
